examples/static_files/src/tests.rs

```diff
@@ -0,0 +1,71 @@
+use std::fs::File;
+use std::io::Read;
+use rocket::testing::MockRequest;
+use rocket::http::Method::*;
+use rocket::http::Status;
+use rocket::response::Body;
+
+use super::rocket;
+
+fn body_to_bytes(body: Body<&mut Read>) -> Option<Vec<u8>> {
```

Comment:
Sorry, I was mistaken. `Body` doesn't implement `Read`. I've added an `into_bytes` method for `Body` that does exactly this.